src/oscar/apps/dashboard/offers/forms.py

```diff
@@ -14,7 +13,14 @@
 class MetaDataForm(forms.ModelForm):
     class Meta:
         model = ConditionalOffer
-        fields = ('name', 'description',)
+        fields = ('name', 'description', 'offer_type')
```

Comment:
Since we don't offer a default implementation for USER and SESSION offers, most likely it is unwanted to offer these as choice to the site administrator, enabling an offer with no effect. Maybe ConditionalOffer.TYPE_CHOICES should only include options that actually have an implementation, or maybe define it based on a setting?

Reply:
@specialunderwear  I have fixed this